RecoJets/JetProducers/python/PileupJetIDParams_cfi.py

```diff
@@ -539,32 +585,7 @@
  label = cms.string("full")
  )
 
-####################################################################################################################  
-simple_5x = cms.PSet(
- impactParTkThreshold = cms.double(1.) ,
- cutBased = cms.bool(False),
```

Comment:
this and similar cleanup goes beyond the scope of the backport of #28984.
This removal appears to apply what was done in #24432 (in 2018)
and is apparently OK to be in this PR.

Reply:
no no, I was not trying to cleanup anything, if you see below, these lines are there. I think it is just some issues with the diff.